crates/forge/bin/cmd/build.rs

```diff
@@ -86,7 +91,7 @@ impl BuildArgs {
 
         // Collect sources to compile if build subdirectories specified.
         let mut files = vec![];
-        if let Some(dirs) = self.args.paths {
+        if let Some(dirs) = self.paths {
             for dir in dirs {
                 files.extend(source_files_iter(dir, MultiCompilerLanguage::FILE_EXTENSIONS));
```

Comment:
dir -> path

and does source_Files_iter work with single files?

Reply:
> and does source_Files_iter work with single files?

yep, from https://docs.rs/walkdir/latest/walkdir/struct.WalkDir.html#method.new:
> If root is a directory, then it is the first item yielded by the iterator. If root is a file, then it is the first and only item yielded by the iterator.